docs/Tutorials/Python.md

```diff
@@ -26,7 +26,12 @@ supports `pip`, for instance in a
 [Anaconda](https://www.anaconda.com/distribution/) environment. You can also get
 access to the SpECTRE Python modules by adding  `BUILD_DIR/bin/python` to your
 `PYTHONPATH`. This is done automatically by sourcing the `LoadPython.sh` file
-with the command `. BUILD_DIR/bin/LoadPython.sh`.
+with the command `. BUILD_DIR/bin/LoadPython.sh`. By default, SpECTRE uses
```

Comment:
I think the `. BUILD_DIR/bin/LoadPython.sh` should print out this information, even with the correct path in it. If that's not printing it, I think that should be what gets fixed, rather than the details here. It might be that CMake scopes `_`-prefixed variables more aggressively than I thought. When I run `. ./bin/LoadPython.sh` on my system I get:
```
You must run python as:
LD_PRELOAD=/usr/lib/x86_64-linux-gnu/libjemalloc.so python ...
```
Did this not happen for you?

Reply:
It did after I ran it. However, as it stands now, running this script is presented as an optional alternative to an alternative in the documentation and I had just pip installed the bindings / set the `PYTHONPATH` manually. Unfortunately the error that python throws is rather cryptic and I naturally assumed I had messed up some of the pybind stuff.

We can give a hint that the path to jemalloc can be printed out with this script.

Reply:
Okay, sounds reasonable.

I have no idea how the pip install should interact with jemalloc, largely because I know nothing about what pip install does so I'm fine with just documenting this here. Switching to `/path/to/libjemalloc.so` is good. You can also note that  near the end of the CMake process it prints out what the `LD_PRELOAD` command should be (I admittedly also don't usually read the CMake output in detail since most of the time it's boring :) ). It might be worth pointing people to that because setting LD_PRELOAD to the _correct_ jemalloc is not so easy (people sometimes install several copies of low-level system packages, anaconda is really terrible for this)